tests/commands/convert/test_build_gui_context.py

```diff
@@ -1,5 +1,5 @@
-def test_overrides_are_not_used(convert_command):
-    overrides = {"gui_framework": "Toga"}
+def test_overrides_are_used(convert_command):
+    overrides = {"gui_framework": "Toga", "console_app": "Console"}
     out = convert_command.build_gui_context({}, overrides)
-    assert out == {"gui_framework": "None"}
-    assert overrides == {"gui_framework": "Toga"}
+    assert out == {"gui_framework": "None", "console_app": "true"}
+    assert overrides["gui_framework"] == "Toga"
```

Comment:
I mentioned this in a previous review, but this is changing the nature of the assertion from a full dictionary to a single key.